src/factory/util/redis.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const redis = require('redis');
+const config = require('painless-config');
+const factoryLogger = require('./logger');
+
+let redisClient = null;
```

Comment:
Need to review this carefully.  I'm not sure that we use the same redis client everywhere.  I recall a case for needing a different one at some point in the code's evolution.  That may have gone away.